plugins/reference-assets/components/src/index.tsx

```diff
@@ -22,9 +29,15 @@ import type {
   InfoAsset,
   InputAsset,
   ImageAsset,
+  ChoiceAsset,
+  ChoiceItem as ChoiceItemType,
 } from "@player-ui/reference-assets-plugin";
 import { dataTypes, validators } from "@player-ui/common-types-plugin";
 
+type WithApplicability<T = Record<string, unknown>> = T & PlayerApplicability;
```

Comment:
Given how often we will need to reuse this, this should probably be a utility that comes from the DSL package. We should also expose a singular type that provides it with the `WithTemplateTypes` type to make it easy to use (`WithPlayerTypes` maybe?)

Reply:
Tools are in https://github.com/player-ui/tools

DSL specifically is in https://github.com/player-ui/tools/tree/main/language/dsl

Reply:
> We should also expose a singular type that provides it with the WithTemplateTypes type to make it easy to use

Could I get some more clarification on this?